warp/handlers/signature_request_p2p_test.go

```diff
@@ -38,7 +38,9 @@ func TestMessageSignatureHandlerP2P(t *testing.T) {
 	backend, err := warp.NewBackend(snowCtx.NetworkID, snowCtx.ChainID, warpSigner, warptest.EmptyBlockClient, database, 100, [][]byte{offchainMessage.Bytes()})
 	require.NoError(t, err)
 
-	msg, err := avalancheWarp.NewUnsignedMessage(snowCtx.NetworkID, snowCtx.ChainID, []byte("test"))
+	offchainPayload, err := payload.NewAddressedCall([]byte{0, 0, 0}, []byte("test"))
```

Comment:
Note this "change" effectively adds the requirement that offchain messages be either addressed call or hash payload